<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListCitedSources</class>
 <widget class="QWidget" name="ListCitedSources">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Cited Sources:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editPart">
       <property name="text">
        <string>edit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="findMore">
       <property name="text">
        <string>find more</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPart">
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePart">
       <property name="text">
        <string>remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
